views/users/view.hbs

```diff
@@ -8,32 +8,181 @@
+          </li>
+          {{/each}}
+        </div>
+
+        <input id="displayAffiliations" type="button" name="expandAffiliations" value="Show past affiliations"
+          onclick="showPastAffiliations()" />
+        {{/if}}
+      </ul>
+      {{/if}}
+
+      <br />
+
+      <strong>Educational background:</strong>
+      <ul class="list">
+        {{#each person.orcidUserData.userEducations}}
+        <li>
+          <b>{{this.organizationName}}</b> - {{this.roleTitle}}
+        </li>
+        {{/each}}
+      </ul>
+
+      <br />
+
     </div>
 
-    <div class="col-sm-3" id="sidebar">
-      <h3>Publications</h3>
+    <div class="col-sm-3">
+      {{#if publications.length}}
+      <canvas id="myChart" width="250px" height="250px"></canvas>
+      {{/if}}
+    </div>
+  </div>
+
+  {{#if publications.length}}
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Publications</h1>
+
+      <hr>
+
       <ul class="list list-unstyled">
-      {{#each publications}}
-        <li><a href="/publications/view/{{this._id}}">{{this.title}}</a></li>
-      {{/each}}
+        {{#each publications}}
+        <li class="publication-container" onclick="window.location='/publications/view/{{this._id}}'">
+          <div class="row">
+            <div class="col-sm-9">
+              <p class="publication-type">
+                {{this.type}}
+              </p>
+              <h4 class="publication-title">
+                {{this.title}}
+              </h4>
+            </div>
+            
+            <div class="col-sm-3">
+              <p>
+                <b>
+                  Date added:
+                </b>
+                {{this.createdDate}}
+              </p>
+            </div>
+          </div>
+        </li>
+        {{/each}}
       </ul>
     </div>
+  </div>
+  {{/if}}
 
+  {{#if person.orcidUserData.userWorks}}
+  <br />
+
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Non-Octopus Publications</h1>
+
+      <hr>
+
+      <ul class="list list-unstyled">
+        {{#each person.orcidUserData.userWorks}}
+
+        <li class="publication-container" onclick="window.location='{{this.url}}'" target="_blank">
+          <div class="row">
+            <div class="col-sm-9">
+              <p class="publication-type">
+                {{this.type}}
+              </p>
+
+              <h4 class="publication-title">
+                {{this.title}}
+              </h4>
+
+              <p class="publication-title">
+                {{this.journalTitle}}
+              </p>
+            </div>
+
+            <div class="col-sm-3">
+              <p>
+                <b>
+                  Date added:
+                </b>
+                {{this.createdDate}}
+              </p>
+            </div>
+          </div>
+        </li>
+        {{/each}}
+      </ul>
+    </div>
   </div>
+  {{/if}}
 </div>
 
 {{> navTop}}
 
 {{> footer}}
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
```

Comment:
Can we store a local copy of this in `/public/vendor` directory?
One DNS lookup fewer :)

Please keep the version of the module in the file name / dir name so that this can be kept up to date (unless we go full npm but that feels unnecessary at this point).